### ISSUE TITLE
Sign-in failed with status code 8 with 0.9.55

### ISSUE THREAD

**Issue by Firemaw** [playgameservices/play-games-plugin-for-unity#2389]
Hi, 

I'm having an issue with sign-in on the new version 0.9.55. I upgraded from 0.9.37 because it was causing exception popups with the new version of Play Games from the Play Store. All the code I'm using is unchanged, which worked previously with 0.9.37. I use a custom config with savegames initialized. But now with 0.9.55 I am having an issue with sign-in as below. Status code 8 is STATUS_APP_MISCONFIGURED "The developer has misconfigured their application in some way. The logs will contain more data about the error and the appropriate resolution." But I can't see any details about the error was what is misconfigured. I've set up achievements and leaderboards in Google Play. Linked all the data and App ID, run the configuration and everything the same as I did for my last game as far as I can see. The Game Services data is currently in Draft mode for an app in Alpha, and I've added myself to the list of Testers.

The AAR versions are 26.1.0 and 15.0.1, which I assume are compatible from the release notes. I'm targeting API 27.

Has anyone else encountered this problem or could help me find what the issue is? I'm scratching my head at the moment. Hopefully it is something simple that I've misconfigured.

Thanks

```
12-07 17:39:46.675 5699-5714/com.comp.proj I/Unity: Starting Auth with token client.
    UnityEngine.DebugLogHandler:Internal_Log(LogType, String, Object)
    UnityEngine.DebugLogHandler:LogFormat(LogType, Object, String, Object[])
    UnityEngine.Logger:Log(LogType, Object)
    UnityEngine.Debug:Log(Object)
    GooglePlayGames.Native.NativeClient:Authenticate(Action`2, Boolean) (at ...\Assets\GooglePlayGames\Platforms\Native\NativeClient.cs:102)
    GooglePlayGames.PlayGamesPlatform:Authenticate(Action`2, Boolean) (at ...\Assets\GooglePlayGames\ISocialPlatform\PlayGamesPlatform.cs:414)
    GooglePlayGames.PlayGamesPlatform:Authenticate(Action`1, Boolean) (at ...\Assets\GooglePlayGames\ISocialPlatform\PlayGamesPlatform.cs:383)
    GooglePlayGames.PlayGamesPlatform:Authenticate(Action`1) (at ...\Assets\GooglePlayGames\ISocialPlatform\PlayGamesPlatform.cs:348)
    GooglePlayGames.PlayGamesLocalUser:Authenticate(Action`1) (at ...
12-07 17:39:46.705 5699-5714/com.comp.proj D/TokenFragment: Fragment exists.. calling processRequests
12-07 17:39:46.705 5699-5714/com.comp.proj D/TokenFragment: Building client for: X (a:false e:false i:false wc:  f: false)
12-07 17:39:46.705 5699-5714/com.comp.proj D/TokenFragment: signIn
12-07 17:39:46.705 5699-5714/com.comp.proj D/TokenFragment: mGoogleSignInClient.silentSignIn
12-07 17:39:46.705 5699-5714/com.comp.proj D/TokenFragment: Done with processRequest, result is pending.
12-07 17:39:46.755 5699-5714/com.comp.proj I/Unity: ---- [0] -- X
    UnityEngine.DebugLogHandler:Internal_Log(LogType, String, Object)
    UnityEngine.DebugLogHandler:LogFormat(LogType, Object, String, Object[])
    UnityEngine.Logger:Log(LogType, Object)
    UnityEngine.Debug:Log(Object)
    Google.Developers.JavaObjWrapper:ConstructArgArray(Object[]) (at ...\Assets\GooglePlayGames\Platforms\Android\Developers\JavaObjWrapper.cs:163)
    Google.Developers.JavaObjWrapper:InvokeCallVoid(String, String, Object[]) (at ...\Assets\GooglePlayGames\Platforms\Android\Developers\JavaObjWrapper.cs:305)
    Com.Google.Android.Gms.Common.Api.PendingResult`1:setResultCallback(ResultCallback`1) (at ...\Assets\GooglePlayGames\Platforms\Android\Gms\Common\Api\PendingResult.cs:55)
    GooglePlayGames.Android.AndroidTokenClient:DoFetchToken(Boolean, Action`1) (at ...\Assets\GooglePlayGames\Platforms\Android\AndroidTokenClient.cs:201)
    Googl
12-07 17:39:47.796 5699-5699/com.comp.proj D/TokenFragment: silentSignIn.onFailure
12-07 17:39:47.796 5699-5699/com.comp.proj E/TokenFragment: Sign-in failed with status code: 8
12-07 17:39:47.796 5699-5699/com.comp.proj E/TokenFragment: Setting result error status code to: 8
12-07 17:39:47.796 5699-5699/com.comp.proj D/TokenPendingResult:  Calling onResult for callback: GooglePlayGames.Android.TokenResultCallback result: Status: Status{statusCode=INTERNAL_ERROR, resolution=null} email: <null> id:<null> access: <null>
```

**Comment by olehkuznetsov** [comment]
Try to link your android app in Play Console  again. Makre sure you are using the SHA1 key from the apk you install on the test device.
 Also check if in Play Console the save games are enabled for you application.

**Comment by olehkuznetsov** [comment]
Just try to add the new one. If the package/sha1 key combination is already exist then it will fail to add it and it's fine.

**Comment by leehans** [comment]
>           Just try to add the new one. If the package/sha1 key combination is already exist then it will fail to add it and it's fine.

Got it, thanks @olehkuznetsov

**Comment by Firemaw** [comment]
After digging some more I found the issue was the manifest in Assets\GooglePlayGames\Plugins\Android\GooglePlayGamesManifest.plugin\AndroidManifest.xml was not being (1) created in the setup process, and (2) merged into the final built manifest. I manually copied the application data (including the app ID) into a manifest worked. I also tried deleting and reinstalling the plugin to attempt to reproduce the error. But after reinstall it was creating and merging the manifests correctly.
